compiler/rustc_middle/src/ty/mod.rs

```diff
@@ -132,7 +133,7 @@ pub struct ResolverOutputs {
     pub maybe_unused_trait_imports: FxHashSet<LocalDefId>,
     pub maybe_unused_extern_crates: Vec<(LocalDefId, Span)>,
     pub reexport_map: FxHashMap<LocalDefId, Vec<ModChild>>,
-    pub glob_map: FxHashMap<LocalDefId, FxHashSet<Symbol>>,
+    pub glob_map: FxHashMap<LocalDefId, StableSet<Symbol>>,
```

Comment:
This also looks like it could stay to be a regular FxHashSet.

Reply:
Done in https://github.com/rust-lang/rust/pull/94188/commits/95ed0262b63ca8d8760970ca04af9d5146c067af, but see compilation log.